09LessonNine/in-class-project/index.html

```diff
@@ -1,10 +1,23 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-  <title>Intel</title>
-  <link rel="stylesheet" href="css/style.css">
+    <title>Gardening</title>
+    <link rel="stylesheet" href="css/css-reset.css">
+    <link rel="stylesheet" href="css/style.css">
 </head>
-<body>
 
+<body>
+    <main>
+        <div id="layer">
+          <p class="top-text"> LANDSCAPING AND MORE </p>
```

Comment:
Avoid adding unnecessary spaces to your textual elements - use CSS to add spacing on either side